app/pages/eventWindUp/EventWindUpCheckYourAnswersPage.scala

```diff
@@ -0,0 +1,32 @@
+
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.eventWindUp
+
+import pages.{CheckAnswersPage, Waypoints}
+import play.api.mvc.Call
+
+case object EventWindUpCheckYourAnswersPage extends CheckAnswersPage {
+  override val urlFragment: String =
+      "event-windup-check-answers"
```

Comment:
Not sure what you mean here Cem, could you clarify?

Reply:
Though internally we're using event 0 to represent windup, in any user-facing code we're using "windup" rather than 0. The fragment is displayed in the browser URL, so I feel it should be kept as windup. The old code that you quote was therefore not right.